pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -430,7 +430,7 @@ static __always_inline int handle_ret_udp_recvmsg(int copied, struct bpf_map_def
     log_debug("kretprobe/udp_recvmsg: pid_tgid: %d, return: %d\n", pid_tgid, copied);
     // segment count is not currently enabled on prebuilt.
     // to enable, change PACKET_COUNT_NONE => PACKET_COUNT_INCREMENT
-    handle_message(&t, 0, copied, CONN_DIRECTION_UNKNOWN, 0, 1, PACKET_COUNT_NONE);
+    handle_message(&t, 0, copied, CONN_DIRECTION_UNKNOWN, 0, 1, PACKET_COUNT_NONE, st->sk);
```

Comment:
This is dereferencing `st` after it has been deleted from the `udp_sock_map`.

Reply:
Fixed